types/event.proto

```diff
@@ -35,4 +35,7 @@ message Event {
 
   // Hooks describes the results of multiple hooks; if event is associated to hook execution.
   repeated Hook hooks = 6 [(gogoproto.nullable) = true];
+  
+  // LastOk displays last time this check was ok; if event status is 0 this is set to timestamp
+  int64 last_ok = 7;
```

Comment:
Let's add `[(gogoproto.customname) = "LastOK"]` to this field, so that we can get something that fits the Go coding standards. (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
Yeah I wasn't sure if this follows the ruby name standard `last_ok` but went with that since the issue specifies. And the `Ok` vs `OK` got me too. Thanks